src/en/models-cmr.md

```diff
@@ -28,12 +17,13 @@ clouds.
 
 CMR addresses the case where one may wish to centralize services within one
 model and share them with disparate models. One can imagine models dedicated to
-tasks such as service monitoring, block storage, and database backends.
+tasks such as service monitoring, block storage, and database backends. Another
+use case would be when you are simply using different cloud types.
 
 The commands related specifically to this subject are:
 
 [`consume`][commands-consume]
-: Adds a remote offer to a model.
+: Adds a remote offer to a model without relating to it.
```

Comment:
Okay, having read some more I think I understand this better now. 'consume' is like, take up the offer, but don't attach it to anything, save it for later? How do you attach to it later? How do you know it exists?

The comments about the description still stand

Reply:
Suggestions welcome :smiley_cat:

Reply:
How about : "Accepts an offer from a different model without immediately adding a relation to it. The offered application will subsequently appear as an application in the current model (e.g. in juju status) and a relation can made as with any other native application."

Or something

Reply:
That's awfully long!

Reply:
Feel free to make it more concise ;)

Reply:
Adds is a word I had difficulty with here - it could be read as the user is "adding" (creating) the offer rather than consuming it. Also, I don't see why the explanation needs to be one line?

Reply:
Because all the other commands have short descriptors. I'd like to avoid long lines here. I agree that it comes off as abrupt since these commands are listed from the outset, before the reader has a chance to digest anything. This is the same approach I took with [storage](https://github.com/juju/docs/blob/master/src/en/charms-storage.md). I asked for the team's opinion on that and the consensus was that it was fine.

Reply:
What about "Links an offer to a model..."

Reply:
It also doesn't help that the `consume` command is the very first one (due to the alphabetical order).

Reply:
What is wrong with "accepts"?

It is fine when the descriptions actually work.